docs/data/migration/migration-pickers-v6/migration-pickers-v6.md

```diff
@@ -268,6 +270,33 @@ You can learn more about the new approach on the [dedicated doc page](https://mu
  </LocalizationProvider>
 ```
 
+### Extend `dayjs` in the Day.js adapter
```

Comment:
Since we have only one plugin impacted, maybe put it in the title for better discoverability and focus only on `customParseFormat`:

````suggestion
### Usage with `customParseFormat` 


The call to `dayjs.extend(customParseFormatPlugin)` has been moved to the `AdapterDayjs` constructor. This allows users to pass custom options to this plugin before the adapter uses it.

If you are using this plugin before the rendering of the first `LocalizationProvider` component and did not call `dayjs.extend` in your own codebase, you will need to manually extend `dayjs`:

```tsx
import dayjs from 'dayjs';
import customParseFormatPlugin from 'dayjs/plugin/customParseFormat';

dayjs.extend(customParseFormatPlugin);
```

The other plugins are still initialized before the adapter initialization.
````